apple/RNCWebView.m

```diff
@@ -235,6 +235,14 @@ - (WKWebViewConfiguration *)setUpWkWebViewConfig
   }
 #endif
 
+#if defined(__IPHONE_OS_VERSION_MAX_ALLOWED) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 140000 /* iOS 14 */
+  if (@available(iOS 14.0, *)) {
+    if (_limitsNavigationsToAppBoundDomains) {
```

Comment:
This variable doesn't seem to be getting set to `true`

Reply:
Thank you! My Obj-C skills are a bit weak, I'll give it another look.

Reply:
nvm. turns out to have been a bug on my end.